src/components/Exercise/ExerciseAdd.js

```diff
@@ -1,32 +1,40 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Button from '../utils/Button'
 import BackButton from '../utils/BackButton'
 import { Styled } from './ExerciseAdd.styles'
 import PropTypes from 'prop-types'
 import { useForm } from 'react-hook-form'
+import { Link } from 'react-router-dom'
 
 ExerciseAdd.propTypes = {
   handleExerciseAdd: PropTypes.func.isRequired,
 }
 
 export default function ExerciseAdd({ handleExerciseAdd }) {
+  const [disabled, setDisabled] = useState(false)
   const { register, handleSubmit, formState } = useForm({ mode: 'onChange' })
 
   return (
     <section>
       <Styled.Wrapper onSubmit={handleSubmit(handleAdd)}>
         <Styled.ButtonWrapper>
           <BackButton />
-          <Button
-            primary
-            mla
-            disabled={
-              !formState.dirty || (formState.dirty && !formState.isValid)
-            }
-            type="submit"
-          >
-            save
-          </Button>
+          {(!disabled && (
+            <Button
+              danger
+              mla
+              disabled={
+                !formState.dirty || (formState.dirty && !formState.isValid)
+              }
+              type="submit"
+            >
+              Save
+            </Button>
+          )) || (
+            <Button primary mla>
```

Comment:
What's primary mla?

Reply:
these are styled component button props for color and a utility prop for margin left auto

Reply:
styled component button props... that's cool!